build.gradle

```diff
@@ -696,6 +696,26 @@ project(':iceberg-dell') {
   }
 }
 
+project(':iceberg-snowflake') {
+  test {
+    useJUnitPlatform()
+  }
+
+  dependencies {
+    implementation project(':iceberg-core')
+    implementation project(':iceberg-common')
+    implementation project(path: ':iceberg-bundled-guava', configuration: 'shadow')
+    implementation project(':iceberg-aws')
```

Comment:
What is used from `iceberg-aws`?

Reply:
Looks like this was vestigial from fiddling around with S3FileIO directly earlier, removed.